etl/steps/data/garden/artificial_intelligence/2023-06-08/yougov_end_of_humanity.meta.yml

```diff
@@ -0,0 +1,425 @@
+dataset:
+  title: AI and the End of Humanity (YouGov, 2023)
+  description: This poll was conducted online on April 7 - 11, 2023 among 1,000 U.S.
+    adult citizens. Respondents were selected from YouGov’s opt-in panel using sample
+    matching.
+  licenses:
+  - name: Public domain
+  sources:
+  - name: YouGov
+    description: This poll was conducted online on April 7 - 11, 2023 among 1,000
+      U.S. adult citizens. Respondents were selected from YouGov’s opt-in panel using
+      sample matching.
+    url: https://today.yougov.com/topics/technology/articles-reports/2023/04/14/ai-nuclear-weapons-world-war-humanity-poll?utm_source=80,000%20Hours%20mailing%20list&utm_campaign=3bb3a9cbc0-EMAIL_CAMPAIGN_2023_05_25_07_19&utm_medium=email&utm_term=0_43bc1ae55c-fcdb5a40fb-%5BLIST_EMAIL_ID%5D
```

Comment:
You can reduce this url to simply:
https://today.yougov.com/topics/technology/articles-reports/2023/04/14/ai-nuclear-weapons-world-war-humanity-poll